x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -342,31 +348,29 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
       cy.get(CONTROL_POPOVER(0)).should('contain.text', 'No options found');
       cy.get(EMPTY_ALERT_TABLE).should('be.visible');
     });
+
     it('should take timeRange into account', () => {
       const startDateWithZeroAlerts = 'Jan 1, 2002 @ 00:00:00.000';
-      const endDateWithZeroAlerts = 'Jan 1, 2010 @ 00:00:00.000';
+      const endDateWithZeroAlerts = 'Jan 1, 2002 @ 00:00:00.000';
```

Comment:
Do you think the previous range was causing an issue?

Reply:
I think I wanted to produce `zero` results. If you notice, start and end date are same.

Reply:
Nit: `startDate...` and `endDate...` only really make sense as a tuple, i.e. `rangeWithZeroAlerts`. Either of those dates by themselves doesn't have much meaning.

Reply:
@rylnd As indicated in [other comment](https://github.com/elastic/kibana/pull/173163#issuecomment-2076436350), these changes are done.